frontend/src/v5/store/projects/projects.helpers.ts

```diff
@@ -27,3 +27,16 @@ export const getProjectImgSrc = (teamspace: string, project: string) => (
 
 export const projectImageFileIsTooBig = (file): boolean => (file.size > clientConfigService.projectImageSizeLimit);
 export const PROJECT_IMAGE_MAX_SIZE_MESSAGE = formatInfoUnit(clientConfigService.projectImageSizeLimit);
+
+export enum ModelTypes {
+	CONTAINER,
+	FEDERATION,
+	DRAWING,
+}
+
+export const getModelType = (model) => {
+	if (model.federate) return ModelTypes.FEDERATION;
+	if (model.units) return ModelTypes.CONTAINER;
```

Comment:
federations also have units, so this can be misleading. Also, if it's not a federation and it's not a drawing, it's always a container, right? If so, could that be the default return value?